src/extension.ts

```diff
@@ -28,8 +28,17 @@ export namespace Openshift {
     export namespace Project {
         export const create = async function createProjectCmd(cli: cli.ICli, explorer: explorerFactory.OpenShiftExplorer, context: odoctl.OpenShiftObject) {
             const value  = await vscode.window.showInputBox({prompt: "Project name"});
-            await cli.execute(`odo project create ${value.trim()}`, {});
-            await explorer.refresh();
+            const characterRegex = /[a-z0-9]([-a-z0-9]*[a-z0-9])?/;
+            if(characterRegex.test(value)) {
+                if(value.length<=63) {
+                    await cli.execute(`odo project create ${value.trim()}`, {});
+                    await explorer.refresh();
+                } else {
+                    await vscode.window.showErrorMessage('Project name is to long');
```

Comment:
We need to use `too` instead of `to`. And we need to inform the user that the limit length for the project name is 63 characters.

Use this message: `Project name should be within 63 characters`.